src/drive/web/modules/navigation/AppRoute.jsx

```diff
@@ -18,6 +18,9 @@ import {
 import { Container as Trash } from 'drive/web/modules/trash'
 import FileHistory from '../../../../components/FileHistory'
 import UploadFromMobile from 'drive/mobile/modules/upload'
+
+import 'cozy-sharing/dist/stylesheet.css'
```

Comment:
Should we import this with the other css files?

Reply:
I'm wondering if we shouldn't move the imports to this file. Currently we're duplicating import in targets/browser/index, targets/mobile/index, public/index. Let me see if I create a new commit to do that ;)

Reply:
Done in https://github.com/cozy/cozy-drive/pull/1940/commits/1b5e09da32733948bc92bf538269d6110c498db8